Small fixes

MatthijsBurgh opened the pull request.

MatthijsBurgh committed:
Append paths at the end of the ruff args

MatthijsBurgh committed:
fix: use provided args

MatthijsBurgh commented:
@DangitBen can you take a look at this PR please?

DangitBen commented:
Sorry, I was out on vacation for a bit. The changes overall look good and like more proper python. For my awareness, what does this aim to fix? It seems like you want to be able to pass arguments into the main function manually? (Possibly from another python script?)

I just want to make sure I can test it as well.

MatthijsBurgh commented:
I hope you enjoyed your vacation.

I changed the order of the arguments of the ruff commands. I am not sure this was really broken. Though my change makes sure, the paths are interpreted as paths and all other arguments as options.

The main function accepted `argv`. Though these were never used by the arg parser. You used the `sys.argv`. Changed it so it used the provided `argv` and changed the default value to `sys.argv[1:]`. Which matches all other ament linting packages.

DangitBen commented:
Okay, that tracks with what I was thinking based on the changes. I've tested it out and it looks good to me! Thanks for the contribution!
